tests/unit/form-reset-mixin/core.js

```diff
@@ -0,0 +1,97 @@
+define( [
+	"jquery",
+	"lib/common",
+	"ui/widget",
+	"ui/form-reset-mixin"
+], function( $, common ) {
+
+module( "widget factory", {
+	setup: function() {
+		$.widget( "ui.testWidget", [ $.ui.formResetMixin, {
+			_create: function() {
+				this._bindFormResetHandler();
+			},
+			_destroy: function() {
+				this._unbindFormResetHandler();
+			},
+			refresh: function() {
+				$.ui.testWidget.refreshed.push( this.element.attr( "id" ) );
+			}
+		} ] );
+
+		$.ui.testWidget.refreshed = [];
+	},
+
+	teardown: function() {
+		if ( $.ui ) {
```

Comment:
Yeah, it's always defined. I had copied this from the widget tests, where it may not exist yet.